build.gradle

```diff
@@ -0,0 +1,56 @@
+buildscript {
+	ext {
+		springBootVersion = '1.5.1.RELEASE'
+	}
+	repositories {
+		mavenCentral()
+	}
+	dependencies {
+		classpath("org.springframework.boot:spring-boot-gradle-plugin:${springBootVersion}")
+	}
+}
+
+plugins {
+    id "com.gorylenko.gradle-git-properties" version "1.4.16"
+}
+
+apply plugin: 'java'
+apply plugin: 'eclipse'
+apply plugin: 'org.springframework.boot'
+
+group = appGroup
+version = appVersion
+
+jar {
+	baseName = appArtifact
+	version = appVersion
+}
+
+repositories {
+	mavenCentral()
+}
+
+dependencies {
+	compile 'org.springframework.boot:spring-boot-starter-actuator'
+	compile 'org.springframework.boot:spring-boot-starter-security'
+	compile 'org.springframework.boot:spring-boot-starter-web'
+
+	runtime 'org.springframework.boot:spring-boot-devtools'
+
+	compileOnly 'org.projectlombok:lombok'
+
+	testCompile 'org.springframework.boot:spring-boot-starter-test'
+}
+
+springBoot  {
+    buildInfo()
+}
+
+processResources {
+    expand(project.properties)
```

Comment:
This tells Gradle when copying resource files that it should replace the token strings (i.e. ${}) with project properties.